anything beyond the most basic introduction should be in the documentation to avoid redundancy

dbu opened the pull request:
fix #47

dbu commented on README.md:
````diff
@@ -29,7 +29,7 @@ Install
 
 ```bash
-$ composer install
+$ composer create-project sculpin/blog-skeleton my-blog
````
this requires the package to be registered with packagist. can one of the maintainers please do that?

dbu replied:
#47 is fixed

dbu commented:
i added some of the things i remove here in https://github.com/sculpin/sculpin.io/pull/119

bellisk commented:
@dbu I noticed that in https://github.com/sculpin/sculpin-blog-skeleton/pull/63 I added more information to the section that you'd already stripped out, oops. Let's look at it quickly together and see how best to arrange it?

dbu commented:
@beryllium i rebased now that the webpack encore thing is merged, and added setup instructions for composer and yarn so its really a complete instructions. i also tested the steps and got a running site :tada:

bellisk commented on README.md:
```diff
@@ -59,77 +73,12 @@ $ php vendor/bin/sculpin generate --watch --server
-installed separately).
+The skeleton provides you with useful configuration and some example data for
+a Sculpin installation. Check out the [Get Started page](https://sculpin.io/getstarted/)
+and have a look atht the [documentation](https://sculpin.io/documentation/).
```
typo: should be 'at the'
